simulation/simple_sensor_simulator/src/sensor_simulation/traffic_lights/traffic_lights_detector.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <simple_sensor_simulator/sensor_simulation/traffic_lights/traffic_lights_detector.hpp>
+
+namespace simple_sensor_simulator
+{
+namespace traffic_lights
+{
+TrafficLightsDetector::TrafficLightsDetector(std::string topic_name, rclcpp::Node & node)
```

Comment:
```suggestion
TrafficLightsDetector::TrafficLightsDetector(const std::string & topic_name, rclcpp::Node & node)
```

Reply:
done in 654a48cde3fb4b639f8b01175fc7745a61f0d587